### PR TITLE
IDEM-57

### PR DESCRIPTION
**Pull request by amagon270**

### CODE REVIEW

**Review by xcjlo985 — APPROVED** [review]
My approval is scoped to the GHAs

File: .github/workflows/eas-build.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    steps:
+      - name: Checkout Branch
+        uses: actions/checkout@v2
+        with:
+          ref: "BUILD"
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.14.2
```
Comment (xcjlo985) [inline .github/workflows/eas-build.yml:35]: The version can be pulled from package.json so it's not specific twice. Refer to the other GHA for steps

**Review by jrocca82 — APPROVED** [review]

File: .github/workflows/eas-build.yml (hunk 1 above)
Comment (amagon270) [reply .github/workflows/eas-build.yml:35]: I tried setting it in package.json but it wasn't working so that's why I specifically installed it here

Comment (xcjlo985) [reply .github/workflows/eas-build.yml:35]: Let's discuss. Would rather fix your problem than regress from the standards

**Pull request merged**